tox.ini

```diff
@@ -1,8 +1,9 @@
-[testenv]
+[tox]
+envlist = py{36,37,38}-{test,build_docs}
```

Comment:
```suggestion
envlist = py{36,37,38}{-build_docs,}
```

because I deleted the thing called test. You could put it back if you wanted?

Reply:
I added it back